security/publish_dispatcher_spec.rb

```diff
@@ -17,8 +17,23 @@
   end
 
   it 'should not be able to access Publish pages as site visitor' do
+    secure_urls = File.readlines('security/secure_public_routes.txt')
+    # check each url
+    secure_urls.each do |url|
+      visit url
+      # check if its a redirect
+      visit response_headers['Location'] if response_headers.key?('Location')
+      # wait for page load
+      page.has_content?('.+')
+      expect(page.status_code).to eq(404)
+    end
   end
 
   it 'should not be able to invalidate Dispatcher cache' do
+    # set http headers to invalidate cache
+    headers = { 'CQ-Handle' => '/content', 'CQ-Path' => '/content' }
+    Capybara.current_session.driver.add_headers(headers)
+    visit('/dispatcher/invalidate.cache')
+    expect(page.status_code).to eq(403)
```

Comment:
Add a note with the link to https://helpx.adobe.com/experience-manager/dispatcher/using/dispatcher-configuration.html saying that documentation says 404, but our testing shows that the page is forbidden instead with response 403 .